utils/rpms/packaging/Makefile_distro_vars.mk

```diff
@@ -27,36 +27,34 @@ SPECTOOL := spectool
 # DISTRO_BASE (i.e. EL_7)
 # from the CHROOT_NAME
 ifeq ($(CHROOT_NAME),epel-7-x86_64)
-DIST        := $(shell rpm $(COMMON_RPM_ARGS) --eval %{?dist})
-VERSION_ID  := 7
-DISTRO_ID   := el7
-DISTRO_BASE := EL_7
-SED_EXPR    := 1s/$(DIST)//p
+DIST         := $(shell rpm $(COMMON_RPM_ARGS) --eval %{?dist})
+VERSION_ID   := 7
+DISTRO_ID    := el7
+DISTRO_BASE  := EL_7
+SED_EXPR     := 1s/$(DIST)//p
+DISTRO_REPOS := base|epel|extras|updates
```

Comment:
Is the actual value of this variable used anywhere?  I'm just seeing a test for it being set or not.

Reply:
The actual value is not currently being used, but it's existence is used to indicate that we have proxies for the distro repos.

Reply:
Removed in current push.